app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="A user-friendly tool to ease the distribution of very large files"
+DESCRIPTION="jigit aims to be a user-friendly wrapper for jigit-mkimage. \
+jigit-mkimage is a simple but fast tool to reconstruct large image files from \
+\".jigdo\" and \".template\" files. \
+It doesn't have any logic to cope with downloading missing files, but will \
+list the missing files that are needed. \
+It is also convenient in case the user has a partially outdated optical media \
+or ISO image containing at least some files which are not outdated."
+HOMEPAGE="https://www.einval.com/~steve/software/JTE/"
+COPYRIGHT="2004-2014 Steve McIntyre
+	2000-2010 George Danchev, Thomas Schmitt"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/download/jigit_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="459285bd743645381912301578316f2bfafb99ce1c10140a01af2b6cb011f193"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	jigit$secondaryArchSuffix = $portVersion
+	cmd:jigdump$commandSuffix = $portVersion
+	cmd:jigit$commandSuffix = $portVersion
+	cmd:jigsum$commandSuffix = $portVersion
+	cmd:jigit_mkimage$commandSuffix = $portVersion
+	cmd:mkjigsnap$commandSuffix = $portVersion
+	lib:libjte$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libjte${secondaryArchSuffix}_devel = $portVersion
+	devel:libjte$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libjte$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:awk
+	cmd:bash
```

Comment:
I think @fbrosson meant that you put those in "REQUIRES" instead of "BUILD_PREREQUIRES" :)